third-party/java/aapt2-proto/BUCK

```diff
@@ -0,0 +1,14 @@
+prebuilt_jar(
+    name = "prebuilt",
+    binary_jar = "aapt2-proto-0.3.1.jar",
```

Comment:
Please, add a readme file indicating where that file was taken from as well as the license file